webClient/src/app/app.component.html

```diff
@@ -46,6 +55,10 @@
 
         <input type="radio" [(ngModel)]="targetMode" name="targetMode" value="PluginFindAnyOrCreate">
         <label for="targetReuse" i18n="reuse any open app@@reuse-any-open">Reuse Any Open</label>
+        <div *ngIf="this.actionType === 'Maximize' || this.actionType === 'Minimize' ">
+          <label class="targetLabel">Target ID</label>
```

Comment:
Similarly we need an i18n string here too. We will find the translations afterward.